seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/SeaTunnelSourceCollector.java

```diff
@@ -97,15 +94,6 @@ public void collect(T row) {
         try {
             if (row instanceof SeaTunnelRow) {
                 String tableId = ((SeaTunnelRow) row).getTableId();
-                int size;
-                if (rowType instanceof SeaTunnelRowType) {
-                    size = ((SeaTunnelRow) row).getBytesSize((SeaTunnelRowType) rowType);
-                } else if (rowType instanceof MultipleRowType) {
-                    size = ((SeaTunnelRow) row).getBytesSize(rowTypeMap.get(tableId));
```

Comment:
we should use row type to init bytes size, it faster.

Reply:
unused

Reply:
SeaTunnelRow will cache the bytes size after init it. It will be used when record metrics in the next stage.

Reply:
You can use `tableSchemas` instead of `rowType`

Reply:
fixed